src/viur/core/db/transport.py

```diff
@@ -0,0 +1,226 @@
+
+
+@deprecated(version="3.8.0", reason="Use 'db.allocate_ids' instead")
+def AllocateIDs(kind_name):
+    """
+    Allocates a new, free unique id for a given kind_name.
+    """
+    if isinstance(kind_name, Key):  # so ein Murks...
+        kind_name = kind_name.kind
+
+    return allocate_ids(kind_name)[0]
+
+
+def get(keys: t.Union[Key, t.List[Key]]) -> t.Union[t.List[Entity], Entity, None]:
+    """
+    Retrieves an entity (or a list thereof) from datastore.
+    If only a single key has been given we'll return the entity or none in case the key has not been found,
+    otherwise a list of all entities that have been looked up (which may be empty)
+    :param keys: A datastore key (or a list thereof) to lookup
+    :return: The entity (or None if it has not been found), or a list of entities.
+    """
+    # accessLog = currentDbAccessLog.get()
+
+    if isinstance(keys, list):
+        # if isinstance(accessLog, set):
+        #     accessLog.update(set(keys))
+
+        resList = list(__client__.get_multi(keys))
+        resList.sort(key=lambda x: keys.index(x.key) if x else -1)
+        return resList
+
+    # if isinstance(accessLog, set):
+    #     accessLog.add(keys)
+
+    return __client__.get(keys)
+
+
+@deprecated(version="3.8.0", reason="Use 'db.get' instead")
+def Get(keys: t.Union[Key, t.List[Key]]) -> t.Union[t.List[Entity], Entity, None]:
+    return get(keys)
+
+
+def put(entities: t.Union[Entity, t.List[Entity]]):
+    """
+    Save an entity in the Cloud Datastore.
+    Also ensures that no string-key with a digit-only name can be used.
+    :param entities: The entities to be saved to the datastore.
+    """
+    if isinstance(entities, Entity):
+        entities = entities,
+
+    # accessLog = currentDbAccessLog.get()
+    # if isinstance(accessLog, set):
+    #     accessLog.update(set(i.key for i in entities if not i.key.is_partial))
+
+    return __client__.put_multi(entities=entities)
+
+
+@deprecated(version="3.8.0", reason="Use 'db.put' instead")
+def Put(entities: t.Union[Entity, t.List[Entity]]) -> t.Union[Entity, None]:
+    return put(entities)
+
+
+def delete(keys: t.Union[Entity, t.List[Entity], Key, t.List[Key]]):
+    """
+    Deletes the entities with the given key(s) from the datastore.
+    :param keys: A Key (or a t.List of Keys) to delete
+    """
+    # accessLog = currentDbAccessLog.get()
+    if isinstance(keys, list):
+        # if isinstance(accessLog, set):
+        #    accessLog.update(set(keys))
+        return __client__.delete_multi(keys)
+
+    # if isinstance(accessLog, set):
+    #     accessLog.remove(keys)
+
+    return __client__.delete(keys)
+
+
+@deprecated(version="3.8.0", reason="Use 'db.delete' instead")
+def Delete(keys: t.Union[Entity, t.List[Entity], Key, t.List[Key]]):
+    return delete(keys)
+
+
+@deprecated(version="3.8.0", reason="Use 'db.utils.is_in_transaction' instead")
+def IsInTransaction() -> bool:
```

Comment:
@ArneGudermann We've this backward method already in the utils. Do we need this twice?